scripts/common.py

```diff
@@ -966,7 +971,10 @@ def managed_elasticsearch_dev_server():
     es_args = ['%s/bin/elasticsearch' % ES_PATH, '-q'] # -q is the quiet flag.
     # Override the default path to ElasticSearch config files.
     es_env = {'ES_PATH_CONF': ES_PATH_CONFIG_DIR}
-    with managed_process(es_args, env=es_env, shell=True) as proc:
+    proc_context = managed_process(
```

Comment:
Hmm I don't think there's an easy way to guard against it, since a string may technically come from any source. The risk comes from strings that come from an untrusted source (user input, file contents, etc.), however these arguments are all constants. I don't see an easy way to "flag" untrusted inputs, so I guess comments are the best option.